tauri/src/endpoints/http.rs

```diff
@@ -14,7 +14,10 @@ pub fn make_request<T: 'static>(
       let response_type = options.response_type.clone();
       request(options).map(
         |response| match response_type.unwrap_or(ResponseType::Json) {
-          ResponseType::Text => format!(r#""{}""#, response),
+          ResponseType::Text => format!(
+            r#""{}""#,
+            response.replace(r#"""#, r#"\""#).replace(r#"\\""#, r#"\""#)
```

Comment:
there's the magick. nice one

Reply:
we may need more of those on the codebase because the way we eval() error messages, will think about a clever way to do it.